### PR TITLE
[bitnami/external-dns] Add RBAC to support F5 VirtualServer external-dns source

### PR DESCRIPTION
**Pull request by mikejoh**
### Description of the change

With these recently merged PRs in upstream:
https://github.com/kubernetes-sigs/external-dns/pull/3503 - RBAC added upstream in `external-dns` helm chart.
https://github.com/kubernetes-sigs/external-dns/pull/3162 - code changes in `external-dns`.

we now need to add the same RBAC to this chart to be able to configure `external-dns` to list F5 `VirtualServer` objects in the cluster.

### Benefits

We'll be able to configure the `--source=f5-virtualserver` flag which instructs `external-dns` to list all `VirtualServer` objects in the cluster, these objects will be used as a source to create DNS records in the given provider.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by mikejoh** [comment]
@javsalgar :wave: would it be possible to review this one? Thanks!

File: bitnami/external-dns/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/containers/tree/main/bitnami/external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 6.15.0
+version: 6.15.1
```
Comment (aoterolorenzo) [inline bitnami/external-dns/Chart.yaml:27]: Adding functionality should imply a minor bump ([Semver summary](https://semver.org/#summary))

```suggestion
version: 6.16.0
```

**Review by aoterolorenzo — CHANGES_REQUESTED** [review]
Thanks for the contribution @mikejoh !

I just left you a minor comment above.

File: bitnami/external-dns/Chart.yaml (hunk 1 above)
Comment (mikejoh) [reply bitnami/external-dns/Chart.yaml]: Ah, of course, fixed here: https://github.com/bitnami/charts/pull/15804/commits/c4d3d4518a93a1d699400ee51ca37a549f2c9061

**Review by aoterolorenzo — APPROVED** [review]
LGTM

**Comment by mikejoh** [comment]
@aoterolorenzo Sorry for pinging in the PR but i just saw that the pipeline triggered on merge to `main` failed: https://github.com/bitnami/charts/actions/runs/4597349311/jobs/8119923921. It's due to:

```
Error: Detected changes in charts without version bump in Chart.yaml. Charts changed: 1 bitnami/external-dns. Version bumps detected: 0
```

Is that expected? :thinking: Let me know if there's something i can do!

**Comment by aoterolorenzo** [comment]
> @aoterolorenzo Sorry for pinging in the PR but i just saw that the pipeline triggered on merge to `main` failed: https://github.com/bitnami/charts/actions/runs/4597349311/jobs/8119923921. It's due to:
> 
> ```
> Error: Detected changes in charts without version bump in Chart.yaml. Charts changed: 1 bitnami/external-dns. Version bumps detected: 0
> ```
> 
> Is that expected? 🤔 Let me know if there's something i can do!

Hi @mikejoh, seems there was a collision when merging two PR's with same version near each other on time. I saw you just bump a minor in #15968 and this indeed have triggered a chart release.

**Pull request merged**